File: Rasmetroncintos/app/src/main/java/com/novacompcr/rasmetroncintos/MainActivity.java
package com.novacompcr.rasmetroncintos;

import android.content.Context;
import android.content.Intent;
import android.os.Bundle;
import android.util.Log;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    Context context;
    private MySQLiteHelper db;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getSupportActionBar().hide();
        context=this;
        db = new MySQLiteHelper(context);
        new android.os.Handler().postDelayed(
            new Runnable() {
                public void run() {
                    Log.i("tag", "pasando de la pantalla de inicio al login");
                    Intent GoLogin = new Intent(context, LoginActivity.class);
                    startActivity(GoLogin);
                }
            },
        500);
        //db.ReinitUser();
        //db.ReinitNotas();
    }
}

File: KotlinEjemplos/Retrofit2/app/src/main/java/com/gsolano/retrofit2/MainActivity.kt
package com.gsolano.retrofit2

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.gsolano.retrofit2.Tiempo.TiempoActivity
import com.gsolano.retrofit2.futbol.FutbolActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        BtnApiTiempo.setOnClickListener(){
            var intent=Intent(this, TiempoActivity::class.java)
            startActivity(intent)
        }


        BtnApiFutbol.setOnClickListener({
            var intent = Intent(this, FutbolActivity::class.java)
            startActivity(intent)
        })

    }
}
